gui/app.py

```
@@ -0,0 +1,583 @@
+                    outputs=[re_video, re_msg, re_files],
+                )
+
+        # Refresh dropdown choices on load
+        app.load(lambda: gr.Dropdown(choices=_project_choices()), outputs=se_project)
+        app.load(lambda: gr.Dropdown(choices=_project_choices()), outputs=re_project)
+
+    return app
+
+
+def launch(
+    share: bool = True,
```

Ruff (0.15.21)

[warning] Boolean-typed positional argument in function definition

(FBT001)